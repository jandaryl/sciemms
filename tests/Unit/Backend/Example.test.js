import { mount } from '@vue/test-utils'
import Component from '@/Counter.vue'

describe('Component', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(Component)
    expect(wrapper.isVueInstance()).toBeTruthy()
    expect(wrapper.element).toMatchSnapshot()
  })
})
