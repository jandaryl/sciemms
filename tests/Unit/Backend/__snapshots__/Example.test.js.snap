// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component is a Vue instance 1`] = `
<div>
  <h1>
    Count: 0
  </h1>
   
  <button
    jest="increment-button"
  >
    +1
  </button>
</div>
`;
